src/common/sorted-table/index.js

```diff
@@ -307,6 +333,12 @@ export default class SortedTable extends Component {
     })
   }, 500)
 
+  _executeGroupedAction = createSelector(
```

Comment:
`_executeGroupedAction` is not a selector.

Can you tell me what is a selector?